dipy/tracking/utils.py

```diff
@@ -503,6 +515,11 @@ def random_seeds_from_mask(mask, seeds_per_voxel=1, affine=None):
     seeds = where + grid - .5
     seeds = asarray(seeds)
 
+    if not seed_count_per_voxel:
+        # Randomize the seeds and select the requested amount
+        np.random.shuffle(seeds)
```

Comment:
I see your point. Currently there is only one dipy function having a random_seed parameter (`dipy.align.sample_domain_regular`). Elsewhere, it is up to the user to set random.seed before calling functions using a random number generator. 

If we add a random seed parameter to functions, the random seed parameter should be uniform throughout the code, and adding a parameter seed=0 as in `dipy.align.sample_domain_regular` could bring a bit a confusion for this particular function. 

What do you guys think? Should we set a standard parameter name to add to function using np.random, say random_seed=0?